source/chapter2-devicetree-basics.rst

```diff
@@ -129,6 +129,8 @@ of characters in :numref:`node-name-characters`.
    ``.``     period
    ``_``     underscore
    ``+``     plus sign
+   ``?``     question mark
+   ``#``     hash
    ``-``     dash
```

Comment:
I'm inclined to change dtc instead unless there are users.

If we don't have some guidance on when to use these characters, then I'd rather not see them added.